### ISSUE TITLE
[Feature request] args optional setup argument

### ISSUE THREAD

**Issue by Malefaro** [nvim-neotest/neotest-go#10]
Hi

Can we get optional setup argument `args` like in [neotest-python](https://github.com/nvim-neotest/neotest-python) to provide extra_args from setup ? 
For example: 
```go
require('neotest-go')({
    args = {"-timeout=60s","-count=1"}
})
```

Because providing them via `neotest.run.run({extra_args={...}})`  forces them to be specified in key bindings, which is not necessary when using neotest in other languages and may conflicts

**Comment by akinsho** [comment]
@Malefaro FYI 👉🏿 https://github.com/nvim-neotest/neotest-go#contributing

**Comment by Malefaro** [comment]
https://github.com/nvim-neotest/neotest-go/pull/11

**Comment by akinsho** [comment]
closed by #11
